Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-2: No Person shall be a Representative who shall not have attained to the Age of twenty five Years, and been seven Years a Citizen of the United States, and who shall not, when elected, be an Inhabitant of that State in which he shall be chosen.
+2: No Person shall be a Representative who shall not have attained to the Age of thirty Years, and been ten Years a Citizen of the State in which he shall be chosen, and who shall not, when elected, be an Inhabitant of that State in which he shall be chosen, or if the State should choose that its Representatives be in service of a specific geographical area within the State, be an Inhabitant of that geographical area.
 
-3: Representatives and direct Taxes shall be apportioned among the several States which may be included within this Union, according to their respective Numbers, which shall be determined by adding to the whole Number of free Persons, including those bound to Service for a Term of Years, and excluding Indians not taxed, three fifths of all other Persons.2  The actual Enumeration shall be made within three Years after the first Meeting of the Congress of the United States, and within every subsequent Term of ten Years, in such Manner as they shall by Law direct. The Number of Representatives shall not exceed one for every thirty Thousand, but each State shall have at Least one Representative; and until such enumeration shall be made, the State of New Hampshire shall be entitled to chuse three, Massachusetts eight, Rhode-Island and Providence Plantations one, Connecticut five, New-York six, New Jersey four, Pennsylvania eight, Delaware one, Maryland six, Virginia ten, North Carolina five, South Carolina five, and Georgia three.
+3: Representatives shall be apportioned among the several States which may be included within this Union, according to their respective Citizen Populations. The actual Enumeration shall be made within three Years after the first Meeting of the Congress of these United States, and within every subsequent Term of five Years, in such Manner as they shall by Law direct. The Number of Representatives shall not exceed one for every thirty thousand Citizens, nor shall it be less than one for every one hundred thousand Citizens, excepting that each State shall have at least one Representative.
 
 4: When vacancies happen in the Representation from any State, the Executive Authority thereof shall issue Writs of Election to fill such Vacancies.
 
-5: The House of Representatives shall chuse their Speaker and other Officers; and shall have the sole Power of Impeachment.
+5: The House of Representatives shall choose their Speaker and other Officers; and shall have the sole Power of Impeachment against officials of the Federal government.
```

Comment:
The Federal government only has authority to impeach its own members, not the members of any state government.